### PR TITLE
Update here-s-what-you-get-with-phone-system.md

### PR DESCRIPTION
**Pull request by devikabhagwats**
I have made the change to explicitly mention that  MOH is not available for Safe and blind transfer in PSDN so that our customers dont assume it is available for those 2 cases.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by devikabhagwats** [comment]
Who need to do copy editing?

**Comment by devikabhagwats** [comment]
Who needs to do copy editing? There are just 2

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Phone System provides the following features. Unless otherwise noted, features a
...
-|Music on hold | Plays default music defined by the service when an external call from the Public Switched Telephone Network (PSTN) is placed on hold. This feature works for one-to-one PSTN-to-Teams calls in addition to calls made to a call queue. This feature provides on-hold notification parity with other platforms. This feature is configurable by the administrator but [currently only via PowerShell](/powershell/module/skype/set-csteamscallingpolicy?view=skype-ps). Hold music is also not supported in consultative transfer of a PSTN call.|
+|Music on hold | Plays default music defined by the service when an external call from the Public Switched Telephone Network (PSTN) is placed on hold. This feature works for one-to-one PSTN-to-Teams calls in addition to calls made to a call queue. This feature provides on-hold notification parity with other platforms. This feature is configurable by the administrator but [currently only via PowerShell](/powershell/module/skype/set-csteamscallingpolicy?view=skype-ps). Hold music is also not supported in consultative transfer,safe transfer and blind transfer of a PSTN call.|
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md:46]:
```suggestion
|Music on hold | Plays default music defined by the service when an external call from the Public Switched Telephone Network (PSTN) is placed on hold. This feature works for one-to-one PSTN-to-Teams calls in addition to calls made to a call queue. This feature provides on-hold notification parity with other platforms. This feature is configurable by the administrator but [currently only via PowerShell](/powershell/module/skype/set-csteamscallingpolicy?view=skype-ps). Hold music is also not supported in consultative transfer, safe transfer, and blind transfer of a PSTN call.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review: @devikabhagwats - please commit my changes. Thanks.

**Pull request closed without merging**